tests/system/python/pair/test_pyton_north_service_pair.py

```diff
@@ -55,7 +55,7 @@ def reset_fledge_local(wait_time):
 
 @pytest.fixture
 def setup_local(reset_fledge_local, add_south, add_north, fledge_url, remote_ip):
-    local_south_config = {"assetName": {"value": remote_south_asset_name}}
+    local_south_config = {"assetName": {"value": local_south_asset_name}}
```

Comment:
Local and remote both same?

Reply:
Remote asset name does not matter. The local set up sends data to remote by http north and then to PI. So if the local asset name is for example asset1 then pi point created will be asset1.